lndr-backend/src/Lndr/Handler/Credit.hs

```diff
@@ -294,3 +297,93 @@ twoPartyBalanceHandler p1 p2 currency = do
     (ServerState pool configTVar _) <- ask
     ucacAddresses <- fmap lndrUcacAddrs . liftIO $ readTVarIO configTVar
     liftIO . withResource pool $ Db.twoPartyBalance p1 p2 (getUcac ucacAddresses currency)
+
+
+multiSettlementHandler :: [CreditRecord] -> LndrHandler NoContent
+multiSettlementHandler transactions = do
+    result <- mapM (\(tx, nonce) -> submitMultiSettlement nonce tx ) $ zip transactions [0..(length transactions)]
```

Comment:
`mapM (uncurry submitMultiSettlement) $ zip [0..(length transactions)]  transactions` is more readable.